### PR TITLE
Version bump to 1.14.5

### PR DESCRIPTION
**Pull request by mrIllo**

### CODE REVIEW

**Comment by glebm** [comment]
Thanks!

**Comment by Ana06** [comment]
What does 1.14.5 change? :thinking: I didn't find a CHANGELOG and I wonder if I should update :smile:

File: assets/javascripts/popper.js [hunk 1]
```diff
@@ -191,7 +192,7 @@ function getOffsetParent(element) {
   var noOffsetParent = isIE(10) ? document.body : null;
 
   // NOTE: 1 DOM access here
-  var offsetParent = element.offsetParent;
+  var offsetParent = element.offsetParent || null;
```
Comment (Ana06) [inline assets/javascripts/popper.js:195]: May I ask what is this for? :thinking: just trying to find out what the new version changes and I am curious why this is needed. :wink:

Comment (glebm) [reply assets/javascripts/popper.js:195]: The changelog is here: https://github.com/FezVrasta/popper.js/releases

All the questions about the actual code should be asked there as well.
This repo simply packages the upstream code as a rubygem.

**Comment by Ana06** [comment]
Thanks! :wink:

**Pull request merged**